server/app/services/export/PdfExporter.java

```diff
@@ -175,7 +177,24 @@ private byte[] buildApplicationPdf(
                 answerData.questionDefinition().getName(),
                 FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
         final Paragraph answer;
-        if (answerData.encodedFileKey().isPresent()) {
+        if (!answerData.encodedFileKeys().isEmpty()) {
```

Comment:
Non-blocking- I know answerData.encodedFileKey().isPresent() and !answerData.encodedFileKeys().isEmpty() mean the same. But is there a preference of one over the other?